### PR TITLE
EL9 compatibility haproxy config updates

### PR DESCRIPTION
**Pull request by swysor**

### CODE REVIEW

File: el8/system/init.d/kazoo-haproxy.redhat [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+#
+# haproxy
+#
+# chkconfig:   - 85 15
+# description:	HAProxy is a free, very fast and reliable solution \
+#				offering high availability, load balancing, and \
+#				proxying for TCP and  HTTP-based applications
+# processname: haproxy
+# config:	   /etc/kazoo/haproxy/haproxy.cfg
+# pidfile:	   /var/run/haproxy/kazoo-haproxy.pid
```
Comment (vlad88sv) [inline el8/system/init.d/kazoo-haproxy.redhat:11]: This seems to be using the old path

File: el9/system/logrotate.d/haproxy.conf [hunk 2]
```diff
@@ -0,0 +1,12 @@
+/var/log/haproxy/haproxy.log {
+    daily
+    rotate 31
+    missingok
+    notifempty
+    compress
+    sharedscripts
+    postrotate
+        /bin/kill -HUP `cat /var/run/syslogd.pid 2> /dev/null` 2> /dev/null || true
```
Comment (vlad88sv) [inline el9/system/logrotate.d/haproxy.conf:9]: it is now `/run/rsyslogd.pid`

File: el8/haproxy/haproxy.cfg [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    ssl-default-server-ciphers ECDH+AESGCM:ECDH+CHACHA20:ECDH+AES256:ECDH+AES128:!aNULL:!SHA1:!AESCCM
+
+defaults
+    mode http
+    retries 3
+    option redispatch
+    timeout connect 5s
+    timeout client 30s
+    timeout server 10s
+    option httpchk GET /
```
Comment (vlad88sv) [inline el8/haproxy/haproxy.cfg:22]: Shouldn't this be also included in el9?

Comment (vlad88sv) [reply el8/haproxy/haproxy.cfg:22]: this missing default makes `haproxy -c -V -f /etc/kazoo/haproxy/haproxy.cfg` complain in EL9 for bigcouch

File: el8/system/init.d/kazoo-haproxy.redhat (hunk 1 above)
Comment (swysor) [reply el8/system/init.d/kazoo-haproxy.redhat:11]: should probably just remove this, we dont use init.d in el8 anywa

File: el8/haproxy/haproxy.cfg (hunk 3 above)
Comment (swysor) [reply el8/haproxy/haproxy.cfg:22]: can you post the error? I didn't see one in rocky9

Comment (vlad88sv) [reply el8/haproxy/haproxy.cfg:22]: it is a warning @crecinos420 any chance you still have a capture or can reproduce to get the warnings we had about this?

File: el9/system/logrotate.d/haproxy.conf [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+        /bin/kill -HUP `cat /var/run/rsyslogd.pid 2> /dev/null` 2> /dev/null || true
```
Comment (vlad88sv) [inline el9/system/logrotate.d/haproxy.conf]: it is now /run/rsyslogd.pid

File: el9/system/logrotate.d/haproxy.conf (hunk 2 above)
Comment (vlad88sv) [reply el9/system/logrotate.d/haproxy.conf:9]: oops wrong line

File: el8/haproxy/haproxy.cfg (hunk 3 above)
Comment (swysor) [reply el8/haproxy/haproxy.cfg:22]: I didn't see the warnings when starting up but I did apply the same configs i did for el8 there and they seem to be working fine now, good catch!

**Review by doogienz — APPROVED** [review]
Excellent smithers!

**Pull request merged**